roscala/src/main/scala/coop/rchain/rosette/prim/tuple.scala

```diff
@@ -88,6 +88,38 @@ object tuple {
     }
   }
 
+  /**
+    * Define the "tuple-safe-nth" primitive.
+    * This safely returns the nth element of the specified Tuple
+    * e.g. (tuple-safe-nth [1 2 3 4 5] 3) ==> 4
+    */
+  object tplSafeNth extends Prim {
+    override val name: String = "tuple-safe-nth"
+    override val minArgs: Int = 2
+    override val maxArgs: Int = 2
+
+    @checkArgumentMismatch
+    override def fn(ctxt: Ctxt): Either[PrimError, RFixnum] = {
+      val elem = ctxt.argvec.elem
+
+      checkTuple(0, elem).map { t =>
+        checkRFixnum(1, elem) match {
+          case Right(n) =>
+            if (n.value < 0)
+              RFixnum(Int.MinValue)
+            else if (n.value < t.elem.size) {
+              val value = t.nth(n.value) match {
+                case None             => RFixnum(Int.MaxValue)
+                case Some(v: RFixnum) => v
+              }
+              value
+            } else
+              RFixnum(Int.MaxValue)
```

Comment:
I resolved some naming conflicts. You should be able to use `Fixnum` from now on.